tests/theme/dark-mode.spec.tsx

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import styled from "styled-components";
+import {
+    Colour,
+    DSThemeProvider,
+    LifeSGTheme,
+    useDSTheme,
+} from "../../src/theme";
+
+// Test component that uses theme colors
+const TestComponent = styled.div`
+    background: ${Colour["bg"]};
+    color: ${Colour["text"]};
+`;
+
+// Component to test the useDSTheme hook
+const ThemeModeDisplay = () => {
+    const { colourMode } = useDSTheme();
+    return <div data-testid="theme-mode">{colourMode}</div>;
+};
+
+describe("Dark Mode Theme", () => {
+    it("should use light mode by default", () => {
+        const { getByTestId } = render(
+            <DSThemeProvider theme={LifeSGTheme.light}>
```

Comment:
based on the unit test description, this should take the theme object without an explicit colour mode?

```suggestion
            <DSThemeProvider theme={LifeSGTheme}>
```